docs/PIPELINES.md

```diff
@@ -520,18 +540,69 @@ being deployed, or vice versa.
 
 If you need to edit and make changes to your deployment pipeline, simply add
 the changes you need to the bottom of your ci.yml located in your
-*-deployments repo. For example, if we have a pipeline we need to edit the
+*-deployments repo to be merged by spruce. 
+For example, if we have a pipeline we need to edit the
 'check_every:' parameter of a resource named git we can add the block below
-under our pipeline layouts:
+under our pipeline:
 
 **ci.yml**
 ```
-layouts:
-  test-sandbox-ops: |+
-    auto *test-sandbox *test-staging
-    test-sandbox -> test-staging
+  (...)
+  layouts:
+    test-sandbox-ops: |+
+      auto *test-sandbox *test-staging
+      test-sandbox -> test-staging
+
+  groups:
+    default:
+    - test-sandbox
+    - test-staging
 
 resources:
 - name: git
   check_every: 15m
 ```
+
+### Useful commands
+
+#### Retrieve **bosh** configuration
```

Comment:
This may no longer be needed, as genesis now grabs the bosh from the exodus data.  Furthermore, the password and cert, even if needed, should come from the concourse integrated vault.  Lets confirm these two things and get updated docs.

Reply:
I've added your remark as a note, with the idea in mind that we also need to document older way to do things for older Genesis installations.